lib/site_prism/exceptions.rb

```diff
@@ -7,4 +7,5 @@ class TimeoutException < StandardError; end
   class TimeOutWaitingForElementVisibility < StandardError; end
   class TimeOutWaitingForElementInvisibility < StandardError; end
   class UnsupportedBlock < StandardError; end
+  NotLoadedError = Class.new(StandardError)
```

Comment:
I would be more inclined to refactor the other error definitions since this way is cleaner :D if @natritmeyer agrees